### DISCUSSION TITLE
How do I get information about the message the user replied to?

### DISCUSSION THREAD

**Discussion by badjano** [python-telegram-bot/python-telegram-bot#2295] [Q&A] [answered]
My bot receives photos from users, but sometimes the user needs to reference the image to do stuff with it, so I was thinking that maybe they could reply the image they sent with information about it, how could I do this? Thanks!

**Comment by Bibo-Joshi** [comment, marked as answer]
The message the incoming message is a reply to is available as `update.(effective)_message.reply_to_message`. See [here](https://python-telegram-bot.readthedocs.io/en/stable/telegram.message.html#telegram.Message.reply_to_message).

**Reply by badjano** [reply]
thanks!
